components/User/UserProfile/PrivatePublicUserProfile/UserDecks.js

```diff
@@ -1,48 +1,116 @@
+                defaultMessage: 'Last updated'
+            }, 
+            sortCreationDate: {
+                id: 'UserDecks.sort.date', 
+                defaultMessage: 'Creation date'
+            },
+            sortTitle: {
+                id: 'UserDecks.sort.title', 
+                defaultMessage: 'Title'
+            }
+        });
+    }
+    getSelectedSort(sortBy){
+        switch(sortBy){
+            case 'timestamp': 
+                return this.context.intl.formatMessage(this.messages.sortCreationDate);
+            case 'title':
+                return this.context.intl.formatMessage(this.messages.sortTitle);
+            case 'lastUpdate': 
+            default: 
+                return this.context.intl.formatMessage(this.messages.sortLastUpdated);
+        }
     }
-
     render() {
+         // define load more results div
+        let loadMoreDiv = '';
+        let meta = this.props.decksMeta;
+        if(meta.links && meta.links.next){
+            let loadMoreContent = <button className="ui button" aria-label='Load more decks' onClick={this.loadMore.bind(this, meta.links.next)}>Load More</button>;
+            if(this.props.loadMoreLoading){
+                loadMoreContent = <div className="ui active text loader">Loading</div>;
+            }
+            if(this.props.loadMoreError){
+                loadMoreContent = 'An unexpected error occurred while fetching more decks';
+            }
+            loadMoreDiv = <div key="loadMoreDiv" className="ui basic segment center aligned">
+                {loadMoreContent}
+            </div>;
+        }
+        let sortBy = meta.sort;
+        let header = (this.props.loggedinuser === this.props.user.uname) 
+            ? <FormattedMessage id='UserDecks.header.myDecks' defaultMessage='My Decks' />
+            : <FormattedMessage id='UserDecks.header.ownedDecks' defaultMessage='Owned Decks' />;
+
         return (
           <div className="ui segments">
             {(this.props.decks === undefined) ? <div className="ui active dimmer"><div className="ui text loader">Loading</div></div> : ''}
             <div className="ui secondary clearing segment">
-                <h1 className="ui left floated header">{(this.props.loggedinuser === this.props.user.uname) ? 'My Decks' : 'Owned Decks' }</h1>
-                <div className="ui right floated pointing labeled icon dropdown button" ref="sortDropdown">
-                    <i className="icon exchange" aria-label="change order" />
-                    <div className="text">Last updated</div>
-                    <div className="menu">
-                        <div className="item active selected" data-value={2}>Last updated</div>
-                        <div className="item" data-value={1}>Creation date</div>
-                        <div className="item" data-value={0}>Title</div>
+                <h1 className="ui left floated header">{header}</h1>
+                { (this.props.decks) && 
```

Comment:
I haven't run the code yet but do you want to do something like: (this.props.decks) ? <html/> : ' '     ?

Reply:
This syntax is equivalent with that you propose. 
See here: https://stackoverflow.com/questions/24502898/show-or-hide-element/39401319#39401319

Reply:
ah nice!